packages/bp/src/core/bpfs/cache-invalidators.ts

```diff
@@ -46,6 +47,11 @@ export namespace CacheInvalidators {
     install(objectCache: ObjectCache) {
       this.cache = objectCache
 
+      // Supports dumb deployments by allow disabling file listerners
+      if (yn(process.env.CORE_DISABLE_FILE_LISTENERS)) {
```

Comment:
Can you make sure to document this env var somewhere in the doc if you consider it to be relevant?

Reply:
I don`t think this is something to be used anywhere else than internally, so that is why I think this shouldn`t be in our docs, thoughts?

Reply:
The minimum is to document it internally (in global.d.ts), technically every variable should be there, and you should use core_env instead of env, so we have typings and can find references in the code

Reply:
ok got your point